### ISSUE TITLE
Not clear how to do "We can call"

### ISSUE THREAD

**Issue by brigj1** [learn-co-curriculum/phase-3-ruby-oo-inheritance-modules#7]
### Canvas Link

https://learning.flatironschool.com/courses/5187/assignments/180355?module_item_id=397998

### Concern

It's not clear to me how/where I should do this:
We can call:

Should I put these commands inside a bin file? 'extending'? 'dance_party'?
Do I run them from 'pry'? irb?

### Suggested Changes

I should probably know how/where to invoke the "We can call" stuff, but, well,...

**Comment by SamuelBanya** [comment]
@brigj1 

This portion of the assignment is slightly vague, but you can basically add that entire block to '/bin/dance_party.rb' if you want to.

Its more of a section that's used to show what you can do with what has been now modified in that assignment so it's not needed for the actual 'learn test' related tests to pass.

**Comment by brigj1** [comment]
Hi Samuel,
Thanks for this note. I'm glad to have some direction, though that comment was from a while ago and it will be a while before I check it out.
My intention in sharing this comment was to say that "a student was confused."
If I was confused, I can imagine that other students might also be confused, but most don't rise to the level of telling you. We don't know what you know. We don't know your intentions except by what you write. Just as you have provided some clarification for me, it might help other students if you work in a version of your reply into the course materials, and then it won't be "slightly vague." Thanks!

**Comment by SamuelBanya** [comment]
Yeah, I am a fellow student as well, so I point stuff like this out in the related lesson pages' 'Issues' pages all the time @brigj1 

I am not an instructor.
